src/Sulu/Bundle/SnippetBundle/Resources/js/views/SnippetAreas/SnippetAreas.js

```diff
@@ -32,7 +32,9 @@ class SnippetAreas extends React.Component<ViewProps> {
         }
 
         this.snippetAreaStore = new SnippetAreaStore(webspace);
-        this.cacheClearToolbarAction = new CacheClearToolbarAction();
+        this.cacheClearToolbarAction = new CacheClearToolbarAction({
+            webspaceKey: webspace,
```

Comment:
I think this should still be optional in cases where we don't have a webspace key.